communication/slack-config/channels.yaml

```
@@ -272,15 +272,19 @@ channels:
     archived: true
   - name: kubernetes-careers
   - name: kubernetes-client
-  - name: kubernetes-contributors
+  - name: kubernetes-org-members
+  - name: kubernetes-new-contributors
     id: C09R23FHP
```

Nit: need alpha sorting 
```suggestion
  - name: kubernetes-new-contributors
    id: C09R23FHP
  - name: kubernetes-org-members    
```

---

Ahh, thanks - I forgot to do that after changing it from maintainers to org members^^;;